frontend/src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/DateSingleValue/DateSingleValue.tsx

```diff
@@ -1,41 +1,116 @@
 import { ConstraintFormHeader } from '../ConstraintFormHeader/ConstraintFormHeader';
 import Input from 'component/common/Input/Input';
 import { parseDateValue, parseValidDate } from 'component/common/util';
+import {
+    ITimezoneSelect,
+    useTimezones,
+} from '../../../../../../hooks/useTimezones';
+import { utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
+
+import { useEffect, useMemo, useState } from 'react';
+import GeneralSelect from '../../../../GeneralSelect/GeneralSelect';
+import { ConditionallyRender } from '../../../../ConditionallyRender/ConditionallyRender';
+import useUiConfig from '../../../../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { styled } from '@mui/material';
+
 interface IDateSingleValueProps {
     setValue: (value: string) => void;
     value?: string;
     error: string;
     setError: React.Dispatch<React.SetStateAction<string>>;
 }
 
+const StyledWrapper = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 8,
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
+
 export const DateSingleValue = ({
     setValue,
     value,
     error,
     setError,
 }: IDateSingleValueProps) => {
+    const { uiConfig } = useUiConfig();
+    const showSelectableTimezone = Boolean(uiConfig.flags.selectableTimezone);
```

Comment:
If I understand correctly: for the first iteration, this will always be false. Is that correct?

Reply:
Yes, the implementation for selectable timezone is there behind the flag until the sdks catch up

Reply:
Got it. But that is for iteration 2, right? Do we want to merge the code to deal with that now, or wait until it's time? (No right or wrong answers; just opening for discussion)